src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/PaginationDecorator.js

```diff
@@ -23,19 +25,27 @@ export default class PaginationDecorator extends React.PureComponent<Props> {
         const {
             total,
             current,
+            loading,
             children,
         } = this.props;
 
         return (
             <section>
                 {!!current && !!total &&
-                    <InfiniteScroller
-                        total={total}
-                        current={current}
-                        onLoad={this.handlePageChange}
-                    >
-                        {children}
-                    </InfiniteScroller>
+                    <div>
+                        <InfiniteScroller
+                            total={total}
+                            current={current}
+                            onLoad={this.handlePageChange}
+                        >
+                            {children}
+                        </InfiniteScroller>
+                        <div className={paginationDecoratorStyles.infiniteScrollLoader}>
```

Comment:
Wouldn't it be safer to show the loader instead of the `InfiniteScroller`? Then there can't be a bug like loading two pages or once, or something like that. And as a goodie the surrounding `div` could be removed 😃